.github/workflows/web-access.yml

```diff
@@ -0,0 +1,17 @@
+name: Web-Access
```

Comment:
I would suggest we provide a more meaningful name. Something like: prod-health. WDYT?

Reply:
sounds good